api/v1beta1/kogitobuild_types.go

```diff
@@ -85,40 +89,53 @@ type KogitoBuildSpec struct {
 	// +optional
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Maven Mirror URL"
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:label"
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:text"
 	MavenMirrorURL string `json:"mavenMirrorURL,omitempty"`
 
 	// Image used to build the Kogito Service from source (Local and Remote).
-	// The operator will use the one provided by the Kogito Team based on the "Runtime" field.
+	//
+	// If not defined the operator will use image provided by the Kogito Team based on the "Runtime" field.
+	//
 	// Example: "quay.io/kiegroup/kogito-jvm-builder:latest".
```

Comment:
maybe use single quote here to avoid \" on crd?

Reply:
The `"` is needed on CRD anyway due to `\n` addition.